src/app/page/cabinet/userDecks/user.decks.component.html

```diff
@@ -50,6 +50,10 @@
           <th>DeckSyntax</th>
           <td>{{selectedDeck.synthax}}</td>
         </tr>
+        <tr>
+          <th>DeckPrice</th>
```

Comment:
The "space" character is missing here and there, but especially here in the user message.